recordedit/recordEdit.app.js

```diff
@@ -263,14 +263,12 @@
                             throw response;
                         });
                     } else if (session) {
-                        var notAuthorizedMessage = "You are not authorized to Update entities.";
                         var notAuthorizedError = new Error(notAuthorizedMessage);
```

Comment:
Since you removed the variable above, this case will fail/error out with no message defined.

Reply:
I added the message to the message map, which will solve this for this case

https://github.com/informatics-isi-edu/chaise/blob/issue%231223/common/utils.js/#L51